src/lipalightninglib.udl

```diff
@@ -42,7 +42,10 @@ interface LightningNode {
     ListActivitiesResponse get_latest_activities(u32 number_of_activities);
 
     [Throws=LnError]
-    Payment get_payment(string hash);
+    IncomingPaymentInfo get_incoming_payment(string hash);
+
+    [Throws=LnError]
+    OutgoingPaymentInfo get_outgoing_payment(string hash);
```

Comment:
Also, it would make sense to return entire activities so that in case this is used to fetch e.g. a swap or reverse swap, the swap/reverse-swap specific info is returned as well